e2e/workspace/MODULE.bazel

```diff
@@ -10,6 +10,7 @@ npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm", dev_dependenc
 npm.npm_translate_lock(
     name = "npm",
     pnpm_lock = "//:pnpm-lock.yaml",
+    run_lifecycle_hooks = False,
```

Comment:
Are all these minor changes just simplifying tests and formatting things?

Reply:
Bingo, just to make the example a bit more performant

Reply:
Why disable all lifecycle hooks? Seems like this might make the e2e test suppress errors that users will see, unless they too set `run_lifecycle_hooks = False` globally, which they probably would not and should not do.

Reply:
as a user, I did add

```python
lifecycle_hooks_envs = {
        "cypress": ["CYPRESS_INSTALL_BINARY=0"],
    },
```

to `npm_translate_lock`, which is probably better for your e2e test than a global disable (assuming one of the performance issues you allude to is cypress downloading its binary files).

Reply:
Oh that's awesome! I'll mirror that here and make a comment in the readme!

Reply:
Add this to this PR and add the comment to the README in this PR or a follow-up?